keanu-project/src/main/java/io/improbable/keanu/algorithms/NetworkSamples.java

```diff
@@ -106,6 +110,18 @@ public double probability(Function<NetworkState, Boolean> predicate) {
         return (double) trueCount / networkStates.size();
     }
 
+    public List<Object> getSample(int sample) {
```

Comment:
Are there any test for these two methods?

Reply:
They are just getters, so I didn't feel the need to